locations/spiders/payless.py

```diff
@@ -0,0 +1,50 @@
+import scrapy
+from locations.items import GeojsonPointItem
+import json
+import re
+import json
+
+default_headers = {
+    "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_1) AppleWebKit/537.36 (KHTML, like Gecko) "
```

Comment:
Does the spider work without a fake user agent? I'd prefer to use a descriptive user agent if possible.

Reply:
Sorry, force of habit :) What kind of user agent would be appropriate? Something like "alltheplacesbot"?

Reply:
Understand. The default is [set here](https://github.com/alltheplaces/alltheplaces/blob/master/locations/settings.py#L20) so you don't need to set anything in your spider.